ballerina/tests/test_client.bal

```diff
@@ -0,0 +1,693 @@
+    boolean aclRuleFound = false;
+    AclRule[]? aclItems = acl.items;
+    if aclItems != () {
+        foreach AclRule acl_item in aclItems {
+            if (acl_item.id == createdAclRule.id) {
+                aclRuleFound = true;
+                break;
+            }
+        }
+    }
+    test:assertTrue(aclRuleFound, "Retrieved ACL does not contain the created ACL rule");
+    Error? res = check client1->/calendars/[<string>createdCal.id]/acl/[<string>createdAclRule.id].delete();
+    test:assertEquals(res, ());
+    res = check client1->/calendars/[<string>createdCal.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config{}
+function testGetCalendarEventInstances() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar createdCal = check client1->/calendars.post(cal);
+    string eventSummary = "Test Event";
+    Event event = {
+        'start: {
+            dateTime: "2023-10-28T03:00:00+05:30",
+            timeZone: "Asia/Colombo"
+        },
+        end: {
+            dateTime: "2023-10-28T03:30:00+05:30",
+            timeZone: "Asia/Colombo"
+        },
+        summary: eventSummary
+    };
+    Event createdEvent = check client1->/calendars/[<string>createdCal.id]/events.post(event);
+    Events instances = check client1->/calendars/[<string>createdCal.id]/events/[<string>createdEvent.id]/instances.get();
+    test:assertNotEquals(instances.items, ());
+    Error? res = check client1->/calendars/[<string>createdCal.id]/events/[<string>createdEvent.id].delete();
+    test:assertEquals(res, ());
+    res = check client1->/calendars/[<string>createdCal.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config{}
+function testMoveCalendarEvent() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar createdCal = check client1->/calendars.post(cal);
+    string summary2 = "Test Calendar 2";
+    Calendar cal2 = {
+        summary: summary2
+    };
+    Calendar createdCal2 = check client1->/calendars.post(cal2);
+    string eventSummary = "Test Event";
+    Event event = {
+        'start: {
+            dateTime: "2023-10-28T03:00:00+05:30",
+            timeZone: "Asia/Colombo"
+        },
+        end: {
+            dateTime: "2023-10-28T03:30:00+05:30",
+            timeZone: "Asia/Colombo"
+        },
+        summary: eventSummary
+    };
+    Event createdEvent = check client1->/calendars/[<string>createdCal.id]/events.post(event);
+    Event moveEvent = check client1->/calendars/[<string>createdCal.id]/events/[<string>createdEvent.id]/move.post(<string>createdCal2.id);
+    test:assertEquals(moveEvent.summary, eventSummary);
+    Error? res = check client1->/calendars/[<string>createdCal2.id]/events/[<string>createdEvent.id].delete();

```

Comment:
Check for other places as well.
```suggestion
    check client1->/calendars/[<string>createdCal2.id]/events/[<string>createdEvent.id].delete();

```

Reply:
Fixed in [5ec2e48](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/134/commits/5ec2e48c2cbc1fa15e119f5cee8d8021de16eae5)